### PR TITLE
[zh-cn] Resync taint-and-toleration page

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

File: content/zh-cn/docs/concepts/scheduling-eviction/taint-and-toleration.md [hunk 1]
```diff
@@ -245,14 +238,14 @@ taint is removed before that time, the pod will not be evicted.
...
 ## Example Use Cases
--->
-## 使用例子
 
-<!--
 Taints and tolerations are a flexible way to steer pods *away* from nodes or evict
 pods that shouldn't be running. A few of the use cases are
 -->
-通过污点和容忍度，可以灵活地让 Pod **避开** 某些节点或者将 Pod 从某些节点驱逐。下面是几个使用例子：
+## 使用例子
```
Comment (Sea-n) [inline content/zh-cn/docs/concepts/scheduling-eviction/taint-and-toleration.md:245]:
```suggestion
## 使用例子   {#example-use-cases}
```

File: content/zh-cn/docs/concepts/scheduling-eviction/taint-and-toleration.md [hunk 2]
```diff
@@ -424,76 +418,66 @@ tolerations:
...
 <!--
 ## Taint Nodes by Condition
--->
-## 基于节点状态添加污点
 
-<!--
 The control plane, using the node {{<glossary_tooltip text="controller" term_id="controller">}},
 automatically creates taints with a `NoSchedule` effect for [node conditions](/docs/concepts/scheduling-eviction/node-pressure-eviction/#node-conditions).
+-->
+## 基于节点状态添加污点
```
Comment (Sea-n) [inline content/zh-cn/docs/concepts/scheduling-eviction/taint-and-toleration.md:462]:
```suggestion
## 基于节点状态添加污点   {#taint-nodes-by-condition}
```

File: content/zh-cn/docs/concepts/scheduling-eviction/taint-and-toleration.md [hunk 3]
```diff
@@ -125,10 +120,9 @@ will tolerate everything.
 
 An empty `effect` matches all effects with key `key1`.
 -->
-{{< note >}}
 存在两种特殊情况：
 
-如果一个容忍度的 `key` 为空且 operator 为 `Exists`，
+如果一个容忍度的 `key` 为空且 `operator` 为 `Exists`，
 表示这个容忍度与任意的 key 、value 和 effect 都匹配，即这个容忍度能容忍任意 taint。
```
Comment (Sea-n) [inline content/zh-cn/docs/concepts/scheduling-eviction/taint-and-toleration.md:126]:
```suggestion
表示这个容忍度与任意的 key、value 和 effect 都匹配，即这个容忍度能容忍任意 taint。
```

File: content/zh-cn/docs/concepts/scheduling-eviction/taint-and-toleration.md [hunk 4]
```diff
@@ -245,14 +238,14 @@ taint is removed before that time, the pod will not be evicted.
...
-## 使用例子
 
-<!--
 Taints and tolerations are a flexible way to steer pods *away* from nodes or evict
 pods that shouldn't be running. A few of the use cases are
 -->
-通过污点和容忍度，可以灵活地让 Pod **避开** 某些节点或者将 Pod 从某些节点驱逐。下面是几个使用例子：
+## 使用例子
+
+通过污点和容忍度，可以灵活地让 Pod **避开** 某些节点或者将 Pod 从某些节点驱逐。
```
Comment (Sea-n) [inline content/zh-cn/docs/concepts/scheduling-eviction/taint-and-toleration.md:247]:
```suggestion
通过污点和容忍度，可以灵活地让 Pod **避开**某些节点或者将 Pod 从某些节点驱逐。
```

File: content/zh-cn/docs/concepts/scheduling-eviction/taint-and-toleration.md [hunk 5]
```diff
@@ -365,17 +360,18 @@ true. The following taints are built in:
...
- * `node.cloudprovider.kubernetes.io/uninitialized`：如果 kubelet 启动时指定了一个 "外部" 云平台驱动，
+* `node.kubernetes.io/not-ready`：节点未准备好。这相当于节点状况 `Ready` 的值为 "`False`"。
+* `node.kubernetes.io/unreachable`：节点控制器访问不到节点. 这相当于节点状况 `Ready`
+  的值为 "`Unknown`"。
+* `node.kubernetes.io/memory-pressure`：节点存在内存压力。
+* `node.kubernetes.io/disk-pressure`：节点存在磁盘压力。
+* `node.kubernetes.io/pid-pressure`: 节点的 PID 压力。
+* `node.kubernetes.io/network-unavailable`：节点网络不可用。
+* `node.kubernetes.io/unschedulable`: 节点不可调度。
+* `node.cloudprovider.kubernetes.io/uninitialized`：如果 kubelet 启动时指定了一个 "外部" 云平台驱动，
```
Comment (Sea-n) [inline content/zh-cn/docs/concepts/scheduling-eviction/taint-and-toleration.md:374]:
```suggestion
* `node.cloudprovider.kubernetes.io/uninitialized`：如果 kubelet 启动时指定了一个 “外部” 云平台驱动，
```

**Review by Sea-n — COMMENTED** [review]
LGTM

File: content/zh-cn/docs/concepts/scheduling-eviction/taint-and-toleration.md (hunk 4 above)
Comment (tengqm) [reply content/zh-cn/docs/concepts/scheduling-eviction/taint-and-toleration.md:247]: Not sure if the stars will work if we remove the space.

**Review by Sea-n — COMMENTED** [review]
/lgtm

**Pull request merged**